parlai/tasks/blended_skill_talk/worlds.py

```diff
@@ -14,6 +14,10 @@
 
 def load_personas(opt):
     print('[ loading personas.. ]')
+    if opt.get('include_personas', True):
```

Comment:
nit, i know this isn't your chagne but: if `not opt.get('include_personas', True)` we shouldn't even waste time loading the personas here.

Reply:
hmm, yea tru i dont want to mess anything tho